pkg/controller/mq/broker/setup.go

```diff
@@ -0,0 +1,91 @@
+package broker
+
+import (
+	"context"
+
+	"k8s.io/client-go/util/workqueue"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/mq"
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	"github.com/crossplane/crossplane-runtime/pkg/event"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+	"github.com/crossplane/crossplane-runtime/pkg/meta"
+	"github.com/crossplane/crossplane-runtime/pkg/ratelimiter"
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+
+	svcapitypes "github.com/crossplane/provider-aws/apis/mq/v1alpha1"
+	awsclients "github.com/crossplane/provider-aws/pkg/clients"
+)
+
+// SetupBroker adds a controller that reconciles Broker.
+func SetupBroker(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter) error {
```

Comment:
I'm getting a linter error related to this line:
```
/provider-aws/pkg/controller/aws.go:179:3: cannot use broker.SetupBroker (value of type func(mgr manager.Manager, l logging.Logger, rl workqueue.RateLimiter) error) as func(manager.Manager, logging.Logger, workqueue.RateLimiter, time.Duration) error value in array or slice literal
```

Looks like we're missing `poll time.Duration` in SetupBroker. See https://github.com/crossplane/provider-aws/blob/3f1b7187455f14c306b3037312a19a791ac9d9eb/pkg/controller/route53resolver/resolverendpoint/hooks.go#L26